### ISSUE TITLE
Cant set wallpaper

### ISSUE THREAD

**Issue by GorlikItsMe** [rocksdanister/lively#159]
**Describe the bug**
When i want select one from default wallpapers i geting error. 
![image](https://user-images.githubusercontent.com/33355238/95476460-8fa32200-0987-11eb-9523-9ea642183c84.png)
Eternal Light

When i set Fluidsv2 I dont get error but wallpaper dont display

**To Reproduce**
1. Open app
2. Click on Fluidsv2
3. Check Desktop
4. Still old normall wallpaper

**Expected behavior**
Should be visible animated background

**Desktop (please complete the following information):**
 - OS:Windows 10 2004

**Additional context**
Another weird thing is this. I have 1 and 3 screan. I think should be 1 and 2
![image](https://user-images.githubusercontent.com/33355238/95476817-f45e7c80-0987-11eb-8d0b-5ded8bf16fca.png)

**Log file**
20201008_130638.txt https://pastebin.com/qYLbNqiu
20201008_130843.txt https://pastebin.com/fvKaG5hB
20201008_165149.txt https://pastebin.com/xxEwVSRP

**Comment by rocksdanister** [comment]
A few things I noticed from the log
1. Your Lively videoplayer is selected as libVLC, since a few versions back I removed it and libMPV(External) is the current recommended video player... so change it (Lively settings -> Wallpaper -> Video Player.)
2. Lively is having some trouble setting the wallpaper, are you running any desktop modification/overlay software along with lively? 
3. Try restarting windows and see if that helps.

Ignore the screen number, it does not matter.
